src/controlcenter.tsx

```diff
@@ -97,31 +97,44 @@ export default class HTML5Visu {
 
     pathConfiguration() : Promise<boolean>{
         return new Promise((resolve)=>{
-        // The request will automatically forwarded to the CoDeSys folder on a PFC. On older controllers we have to forward to /PLC manually
-        // A first try for get a manually forwarding
-            fetch(this.rootDir+'/visu_ini.xml')
+            // Get the current path
+            let path = window.location.pathname.replace("/webvisu.html", "");
```

Comment:
Great idea. Is window.location.pathname handling the string always in lower case?

Reply:
normally yes, unless the original URL need a uppercase somewhere.
Since we don't know what future case we will have I think it is easier to keep it like the original.
If we have access to the webvisu.html we should have access to the other webvisu files.

Reply:
Okay, I have set the output filenames to lowercase (e.g. webvisu.html, webvisu.js)